### ISSUE TITLE
sends and receives object properties

### ISSUE THREAD

**Issue by cameronmore** [CommonCoreOntology/CommonCoreOntologies#293]
I have a few thoughts about these properties, and I would like to hear others' thoughts as well.

1. Sending and receiving seem be more broad than just communication messages--sending and receiving packages, mail, supplies, and so on. I think what unites these phenomena is the intentional, directed movement of material entities to a pre-known destination.
2. It is not clear to me what it means to send an information content entity. ICEs are always borne by some material--whether it's a computer, piece of paper, portion of the brain, or sound waves through the air. To send an ICE sounds like some more complicated process involving the movement or change of some material substrate. The definitions make reference to 'encoding' and 'decoding' ICEs, but something more complicated seems to be going on. There is the production of the ICE, then the concretizing of it in some material bearer, and then there is the physical movement or distribution of that bearer.
3. Sending and receiving are in themselves processes that should not be collapsed. I can send a letter or a message and it may never reach its destination, similarly with speaking and hearing. Do these warrant their own acts? Act of Sending? Act of Receiving?

Perhaps it's a labeling issue, but since sends/receives are tied to processes of communication, it is more difficult to coin labels for supply chain processes since sending and receiving themselves are very general in natural language.

**Comment by swartik** [comment]
Those properties always struck me as at odds with the other `participates in` subproperties. I don't mean contradictory, just that the other subproperties describe some type of participation in a process, whereas sends and receives are more about participating in a specific type of process (`Act of Communication`). As such I would think that `sends` should be a subproperty of `agent in` and `receives` a subproperty of `is affected by`.

Before I offer any more opinions, though, I'd like to hear from someone who knows the history of these properties. Were they included to support some need?

**Comment by cameronmore** [comment]
Yeah I agree the language of encoding is very agential and decoding is very affected. I think swapping out 'encoding' with 'concretizing' helps me envision what processes are occurring--like as the sender of a message, I'm taking an ICE from my thoughts and concretize it in some material entity which then changes in location, and due to the nature of communication, the receiver is affected by the content
